### ISSUE TITLE
Handle secrets from files

### ISSUE THREAD

**Issue by keithy** [cakephp/phinx#1526]
Docker encourages the use of files for secrets. I always make my PDO class compatible with phinx environments as far as possible. I recently added:
```
$password = $password ?? $config['pass'] ?? trim(get_file_contents($config['pass_file']));;
```
I wondered whether phinx might be interested in doing likewise?

K.

**Comment by lorenzo** [comment]
you can configure phinx using a php file, which allows you to read any secrets from any place you need

**Comment by keithy** [comment]
indeed, that's what I do anyway, but for others that may wish to use the yml configuration it still makes sense to me.
